spec/models/user_spec.rb

```diff
@@ -2220,6 +2220,16 @@ def filter_by(method)
     end
   end
 
+  describe "Destroying a user with security key" do
+    let!(:security_key) { Fabricate(:user_security_key_with_random_credential, user: user) }
+    fab!(:admin) { Fabricate(:admin) }
```

Comment:
Isn't `admin` fabricated "earlier"?

Reply:
not in a parent block, there are other describe at the same hierarchy level with an admin being fabricated, but that would work right?